From: Stefano Rivera <stefanor@debian.org>
Date: Sun, 7 Nov 2021 15:04:10 -0800
Subject: Skip tests for Windows launchers

We don't include the bundled simple_launcher .exes in our Debian source
package, for DFSG reasons. They aren't needed outside Windows.

Forwarded: not-needed
---
 tests/test_scripts.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/test_scripts.py b/tests/test_scripts.py
index 2da6577..6442e19 100644
--- a/tests/test_scripts.py
+++ b/tests/test_scripts.py
@@ -40,6 +40,8 @@ def _read_launcher_data(section, kind):
         return f.read()
 
 
+@pytest.mark.skip(
+    "Skipped on Debian, we don't include the simple_launcher .exes")
 @pytest.mark.parametrize("section", ["console", "gui"])
 @pytest.mark.parametrize("kind", ["win-ia32", "win-amd64", "win-arm"])
 def test_script_generate_launcher(section, kind):
@@ -60,6 +62,8 @@ def test_script_generate_launcher(section, kind):
     assert b"baz.qux()" in code
 
 
+@pytest.mark.skip(
+    "Skipped on Debian, we don't include the simple_launcher .exes")
 @pytest.mark.parametrize(
     "section, kind",
     [("nonexist", "win-ia32"), ("console", "nonexist"), ("nonexist", "nonexist")],
